packages/react-variant-view/src/util/FilterUtils.ts

```diff
@@ -0,0 +1,199 @@
+// this component comes from signal
+import { CancerTypeFilter, DataFilter } from 'react-mutation-mapper';
+
+import {
+    IGeneFrequencySummary,
+    ITumorTypeFrequencySummary,
+} from '../model/GeneFrequencySummary';
+import {
+    IExtendedMutation,
+    IMutation,
+    ITumorTypeDecomposition,
+} from '../model/Mutation';
+
+export const HUGO_SYMBOL_SEARCH_FILTER_ID = '_signalHugoSymbolSearchFilter_';
+export const HUGO_SYMBOL_DROPDOWN_FILTER_ID =
+    '_signalHugoSymbolDropdownFilter_';
+export const CANCER_TYPE_FILTER_ID = '_signalCancerTypeFilter_';
+export const MUTATION_STATUS_FILTER_ID = '_signalMutationStatusFilter_';
+export const PROTEIN_IMPACT_TYPE_FILTER_ID = '_signalProteinImpactTypeFilter_';
+export const PENETRANCE_FILTER_ID = '_signalPenetranceFilter_';
+export const HUGO_SYMBOL_FILTER_TYPE = 'signalHugoSymbol';
+export const MUTATION_STATUS_FILTER_TYPE = 'signalMutationStatus';
+export const MUTATION_COUNT_FILTER_TYPE = 'signalMutationCount';
+export const PENETRANCE_FILTER_TYPE = 'signalPenetrance';
+
+export enum MutationStatusFilterValue {
```

Comment:
At this point we only need `MutationStatusFilterValue` from this file, right? Maybe add a new model into `cbioportal-utils` named `SignalMutationStatus` and use that instead? We can update signal to use the new model, too.